### PR TITLE
Pypsutil changes

### PR DESCRIPTION
**Pull request by camcunningham**
Changed instances of psutil to pypsutil and commited the dist folder with the WHL file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -47,7 +47,7 @@ notebooks/.ipynb_checkpoints/
 # Packaging files
 .tox
 *.egg-info/
-dist/
+# dist/
```
Comment (haz) [inline .gitignore:50]: Why this change?

**Review by haz — COMMENTED** [review]
This all seems really minimal. Let's make sure we deal with `dist/` correctly, and then we can merge here and put a PR in for the core `tarski` repo.

File: .gitignore (hunk 1 above)
Comment (camcunningham) [reply .gitignore:50]: Uncommented so that we have access to the WHL file, otherwise it doesn't get committed. Potentially just download and send you the WHL file instead of committing it?

Comment (haz) [reply .gitignore:50]: Ya, let's leave the changes minimal, and manually generate the whl if need be on our own side. If the base tarski project adopts the changes, then it wouldn't be required, right? Pyodide could just take the pip version of the lib.

Comment (camcunningham) [reply .gitignore:50]: Right. That's a great point. I'll remove that change and send the WHL / have you generate it.

**Comment by camcunningham** [comment]
@haz Check it out now, reverted that change.
